tests/wpt/metadata-css/css-backgrounds-3_dev/html4/attachment-local-clipping-image-6.htm.ini

```diff
@@ -0,0 +1,3 @@
+[attachment-local-clipping-image-6.htm]
```

Comment:
It's not really a regression, just a failure that was hidden by the bug this PR fixes. I think I know what the issue is though and hope to fix it in a later PR.